form_designer/tests/test_basics.py

```diff
@@ -87,6 +87,7 @@ def test_simple_form(
 
         # Test that the form log was saved:
         flog = FormLog.objects.get(form_definition=fd)
+        assert flog == context['form_log']  # (and it's the same object in the context)
```

Comment:
Yeah, but that was a continuation to the comment on line 88 which isn't (and can't nicely be) an assert comment.